holoviews/pyodide.py

```diff
@@ -74,6 +74,10 @@ def render_png(element):
 def render_svg(element):
     return render_image(element, 'svg')
 
+def in_jupyterlite():
+    import js
+    return hasattr(js, "_JUPYTERLAB")
```

Comment:
Looks a little hackish...is there no public/official way to tell what your environment is?

Reply:
I agree, but I can't seem to find a better way to do it. Maybe it just needs a fresh set of eyes; you can give it a try [here](https://jupyter.org/try-jupyter/lab).

Reply:
Jupyter has always made this sort of thing needlessly difficult - I wish they would allow a public API and honestly, I'm impressed you found anything that works!